config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml

```diff
@@ -60,23 +60,25 @@ spec:
                 same as the name returned by the CSI GetPluginName() call for that
                 driver. Required.
               type: string
-            snapshotClassName:
-              description: name of the SnapshotClass to which this snapshot belongs.
-              type: string
             source:
               description: source specifies from where a snapshot will be created.
                 This field is immutable after creation. Required.
               properties:
                 snapshotHandle:
-                  description: snapshotHandle specifies the CSI name of a pre-existing
-                    snapshot on the underlying storage system. This field is immutable.
+                  description: snapshotHandle specifies the CSI "snapshot_id" of a
+                    pre-existing snapshot on the underlying storage system. This field
+                    is immutable.
                   type: string
                 volumeHandle:
-                  description: volumeHandle specifies the CSI name of the volume from
-                    which a snapshot should be dynamically taken from. This field
+                  description: volumeHandle specifies the "volume_id" of the volume
```

Comment:
sorry, also specify "CSI" here too (and everywhere else snapshot_id, or volume_id) is used